common/k8s/src/kube_stats/cluster_stats.rs

```diff
@@ -0,0 +1,70 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub struct ClusterStats {
+    pub resource: String,
+    pub r#type: String,
+    pub containers_init: i32,
```

Comment:
signed, no unsigned?

Reply:
changed the ones that aren't mirroring the types from the kube lib.